### ISSUE TITLE
Allow underscores to be considered as well in parameterize

### ISSUE THREAD

**Issue by wout** [cadmiumcr/transliterator#2]
I'm running into an issue with the following string using the parameterize method: `__important__`. It remains untouched. Is that intentional?

I would expect the leading and trailing underscores to be stripped. Also, I would love to see the option to convert underscores to dashes as well:

```crystal
Cadmium::Transliterator.parameterize("__Important__ message", convert_underscores: true)
# => "important-message"
```

I wouldn't mind creating a PR.

**Comment by watzon** [comment]
So having reviewed `ActiveSupport::Inflector`'s behavior this is intentional, but I would be happy to accept a PR for a `convert_underscores` parameter.

**Comment by wout** [comment]
Great, I'll prepare a PR.

**Comment by watzon** [comment]
Fixed in #3
